go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/casbin/caddy-authz
```

Comment:
Here are should `github.com/casbin/caddy-authz/v2`

Reply:
OMG...Thanks,I don't know it before...

Reply:
@nodece